aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -58,17 +58,22 @@ module aptos_framework::aptos_account {
     }
 
     /// Batch version of APT transfer.
-    public entry fun batch_transfer(source: &signer, recipients: vector<address>, amounts: vector<u64>) {
+    public entry fun batch_transfer(
+        source: &signer, recipients: vector<address>, amounts: vector<u64>
```

Comment:
Why should this be on a separation line if the previous one hasn't exceeded the limit?

Reply:
Since we treat the parameter list separately, if we include the length of the parameter list, it exceeds the line width limit.